rhizome/host/bin/prep_host.rb

```diff
@@ -85,3 +85,6 @@
 r "apt-get -y install qemu-utils mtools"
 
 SpdkSetup.prep
+
+# cron job to store serial.log files
+FileUtils.mkdir_p("/var/log/ubicloud/serials")
```

Comment:
Should we run it on the existing vm hosts before deployment? Otherwise, the GithubRunner deprovisioning will get stuck.

Reply:
Yes, I will do that.